src/legacy/core_plugins/markdown_vis_type/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "markdown_vis_type",
```

Comment:
If we named "vis-type" plugins as `vis_type_markdown` instead, they would all be in one place in folder when sorted alphabetically. What do you think?